From 82fea323c009a037b8cb88cbe7045547f75e1411 Mon Sep 17 00:00:00 2001
From: Xie XiuQi <xiexiuqi@huawei.com>
Date: Fri, 26 Feb 2021 20:21:08 +0800
Subject: [PATCH] arm64/mpam: don't reserve mon 0, we can use it as nomarl
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 82fea323c009a037b8cb88cbe7045547f75e1411


hulk inclusion
category: feature
feature: ARM MPAM support
bugzilla: 48265
CVE: NA

--------------------------------

Signed-off-by: Xie XiuQi <xiexiuqi@huawei.com>
Reviewed-by: Hanjun Guo <guohanjun@huawei.com>
Signed-off-by: Yang Yingliang <yangyingliang@huawei.com>
Signed-off-by: Wang ShaoBo <bobo.shaobowang@huawei.com>
Reviewed-by: Cheng Jian <cj.chengjian@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/kernel/mpam_mon.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm64/kernel/mpam_mon.c b/arch/arm64/kernel/mpam_mon.c
index d976d8613c2c..4e5f64fda50a 100644
--- a/arch/arm64/kernel/mpam_mon.c
+++ b/arch/arm64/kernel/mpam_mon.c
@@ -87,9 +87,6 @@ void mon_init(void)
 	}
 
 	mon_free_map = BIT_MASK(num_mon) - 1;
-
-	/* pmg 0 is always reserved for the default group */
-	mon_free_map &= ~1;
 }
 
 int alloc_mon(void)
-- 
2.26.2

